versioned_docs/version-5.x/screen-tracking.md

```diff
@@ -18,35 +18,41 @@ This example shows how to do screen tracking and send to Firebase Analytics usin
  <samp id="screen-tracking-for-analytics" />
 
 ```js
-import * as React from 'react';
-import * as Analytics from 'expo-firebase-analytics';
+import { useRef } from 'react';
+import analytics from '@react-native-firebase/analytics';
```

Comment:
I'd like to keep `expo-firebase-analytics` in here because is the only firebase analytics library that has universal support for react-native apps ( so far that I know )

Reply:
Ok then!
Rebased and resolved it!